module/power_domain/src/mod_power_domain.c

```diff
@@ -930,10 +944,15 @@ static void process_set_state_request(
     }
 
     if (pd_in_charge_of_response != NULL) {
+#ifdef BUILD_HAS_NOTIFICATION
```

Comment:
this is not required. fwk_thread_get_delayed_response is separate than notifications

Reply:
yes